cachi2/core/package_managers/yarn_classic/workspaces.py

```diff
@@ -60,7 +60,7 @@ def _get_workspace_paths(workspaces_globs: list[str], source_dir: RootedPath) ->
     """Resolve globs within source directory."""
```

Comment:
> which could lead to errors or
> unexpected behavior during package resolution (for example, when
> checking if a workspace contains a package.json file).

Do you have a specific error example/scenario for better illustration?

Reply:
We found the error when processing [this](https://github.com/openshift/console/blob/release-4.14/frontend/packages/.eslintrc.js) (I linked the file that causes the issue).